src/libs/actions/User.js

```diff
@@ -0,0 +1,12 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+function getBetas() {
+    Onyx.merge(ONYXKEYS.BETAS, []);
```

Comment:
This shouldn't be necessary. Is there a reason you added it?

Reply:
So the default is an empty array and not undefined? Makes it easier to use later, as you don't need to do `betas && betas.indexOf('aBeta')`

Reply:
The value on this key is getting overwritten almost immediately after the API call so I don't think this is necessary, and I don't think the problem you mentioned is an actual problem, or if it's a real problem, it needs solved in a different way.

All code that works with Onyx should absolutely work with undefined values for keys. In fact, this is what allows `defaultProps` to work.

Also, I think that `merge()` might not be doing anything in this case. It's like you're trying to merge an empty array into what is already existing in the key, which I assume just does nothing.

Reply:
Good points, I'll do some tests and remove it if all is good